src/AspNet.Security.OAuth.Introspection/Events/IntrospectTokenContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json.Linq;
+
+namespace AspNet.Security.OAuth.Introspection {
+    public class IntrospectTokenContext : BaseIntrospectionContext {
```

Comment:
Yeah. Do you think the naming should be clearer? Something like `InitiateIntrospectionRequestContext` or similar?